worker_tasks/feed_scraper.py

```diff
@@ -17,12 +19,16 @@
 celery = Celery(app.name, broker=app.config['CELERY_BROKER_URL'])
 celery.conf.update(app.config)
 
+
+def get_full_text(url):
+    html = requests.get(url).content
+    return Document(html).summary()
```

Comment:
Can you link to documentation for this? Trying to find it online and having trouble.

For example, what does .summary() do and why isn't it something like .get_text() ?